main.py

```diff
@@ -75,6 +78,13 @@ def export_streams_to_clipboard(self):
         clipboard.clear(mode=clipboard.Clipboard)
         clipboard.setText(text, mode=clipboard.Clipboard)
 
+    def import_streams_from_clipboard(self):
```

Comment:
i figured that we would take it from the clipboard because that what it said in musses [commit](https://github.com/kaszim/desktop-stream-viewer/pull/84/files#diff-572c860eeb51df20cb79afd9012d9184R998)? Since I do not know of any other import feature than this i don't see the need to be more generic, but it certainly doesn't hurt 👍